src/com/lilithsthrone/game/dialogue/DebugDialogue.java

```diff
@@ -347,6 +349,54 @@ public void effects() {
 						}
 					};
 					
+				} else if (index == 8) {
+					return new Response("Lilaya's tests", "Automatically completes Lilaya's enchantment quest, making you able to enchant right away.", DEBUG_MENU){
+						@Override
+						public void effects() {
+							if(!Main.game.getPlayer().isQuestCompleted(QuestLine.SIDE_ENCHANTMENT_DISCOVERY)){ //If the player hasn't completed the enchantment quest
+								if(Main.game.getPlayer().hasQuest(QuestLine.SIDE_ENCHANTMENT_DISCOVERY)){ //But has started it
+									Main.game.getTextEndStringBuilder().append(Main.game.getPlayer().setQuestProgress(QuestLine.SIDE_ENCHANTMENT_DISCOVERY, Quest.SIDE_UTIL_COMPLETE)); //Finish it
+								}
+								else{ //But hasn't started it
+									Main.game.getTextEndStringBuilder().append(Main.game.getPlayer().startQuest(QuestLine.SIDE_ENCHANTMENT_DISCOVERY)); //Start the quest
+									Main.game.getTextEndStringBuilder().append(Main.game.getPlayer().setQuestProgress(QuestLine.SIDE_ENCHANTMENT_DISCOVERY, Quest.SIDE_UTIL_COMPLETE)); //And finish it
+								}
+							}
+						}
+					};
+
+				} else if (index == 9) {
+					return new Response("+1 Epona stamp", "Gives you one stamp you can get by playing the breeder roulette.", DEBUG_MENU){
+						@Override
+						public void effects() {
+							Main.game.getDialogueFlags().eponaStamps += 1;
+							Main.game.getTextEndStringBuilder().append("Added 1 stamp, you now have "  + Main.game.getDialogueFlags().eponaStamps + " stamp(s)");
+						}
+					};
+
+				} else if(index == 10){
+					return new Response("Get slaver license", "Automatically completes the quest to get a slaver license. This will start the quest if you don't already have it, and finish it.", DEBUG_MENU){
+						@Override
+						public void effects(){
+							if(!Main.game.getPlayer().isHasSlaverLicense()){ //If the player doesn't have the slaver license
+								if(Main.game.getPlayer().hasQuest(QuestLine.SIDE_SLAVERY)){ //But has started the quest to get it
+									Main.game.getTextEndStringBuilder().append(Main.game.getPlayer().setQuestProgress(QuestLine.SIDE_SLAVERY, Quest.SIDE_UTIL_COMPLETE)); //Finish it.
+								}
+								else{ //But hasn't started the quest to get it
+									Main.game.getTextEndStringBuilder().append(Main.game.getPlayer().startQuest(QuestLine.SIDE_SLAVERY)); //Start the quest
+									Main.game.getDialogueFlags().values.add(DialogueFlagValue.finchIntroduced); //Introduce Finch
+									Main.game.getTextEndStringBuilder().append(Main.game.getPlayer().setQuestProgress(QuestLine.SIDE_SLAVERY, Quest.SIDE_UTIL_COMPLETE)); //And finish it
+								}
+							}
+						}
+					};
+				} else if (index == 11) {
+					return new Response("Get KKP", "Adds an extremely overpowered weapon in your inventory.", DEBUG_MENU) {
```

Comment:
I think a better solution would be to just remove the ItemTag.REMOVE_FROM_DEBUG_SPAWNER from the KKP. However, since Inno deliberately added that tag quite recently (updating from older special-case code), it seems like she wants it to remain hidden.

https://github.com/Innoxia/liliths-throne-public/blob/ae1a4928a35501c8f8bd895c32f98c2d9487336e/src/com/lilithsthrone/game/inventory/weapon/WeaponType.java#L676-L678

Reply:
Personally, I have it un-hidden in my own private builds. I'd support either direction, but it is Inno's call at the end of the day.